packages/cli/src/services/project-initializer.ts

```diff
@@ -40,6 +40,15 @@ export async function generateRootDirectory(config: ProjectInitializerConfig): P
   await Promise.all(dirs.map(createDirectory))
 }
 
+export async function initializeGit(config: ProjectInitializerConfig): Promise<void> {
+  try {
+    await exec('git init', { cwd: projectDir(config) })
+    await exec('git add -A && git commit -m "Initial commit"', { cwd: projectDir(config) })
```

Comment:
Why not `git init && git add -A && git commit -m "Initial commit"`?

Reply:
And fixed this too, forgot to merge them before.